docs/release-notes/2.13.0.md

```diff
@@ -0,0 +1,68 @@
+
+    > Note: This capability is being released as a **Technical Preview**. We do not recommend the use of Integrated Alerting in enterprise or production environments until the functionality is released as general availability (GA). While in Technical Preview status, the feature is not supported by Percona Support SLA (but is supported by Product/Engineering on a best-effort basis).
+
+2. [Ability to monitor SSL-enabled MongoDB](https://jira.percona.com/browse/PMM-5364)
+
+    Allows PMM administrators to set up configured SSL certificate “keys” to authenticate the connection to PMM, specifically for setting up MongoDB. This is a critical security requirement especially in large enterprise infrastructure environments.
+
+3. [Node Summary/Nodes Overview dashboards: Show External services on dashboards](https://jira.percona.com/browse/PMM-6713) (Technical Preview)
+
+    Improves the user experience for adding and viewing external services on the Node Summary dashboard of PMM. External services means any data that can be monitored by a Prometheus export, for example, non-Percona supported databases like Reddis, ElasticSearch, Cassandra, etc. or an organization’s external application.
+
+4. DBaaS Preview phase 1.0 (Technical Preview). We are also releasing the first preview of DBaaS functionality. You can read more about [configuration](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/dbaas.html) and [usage](https://www.percona.com/doc/percona-monitoring-and-management/2.x/using/platform/dbaas.html).
+
+
+
+## Improvements
+
+* [PMM-5364](https://jira.percona.com/browse/PMM-5364): Ability to monitor SSL-enabled MongoDB (Thanks to Hubertus Krogmann for reporting this issue)
+* [PMM-7086](https://jira.percona.com/browse/PMM-7086): VictoriaMetrics end-points replace Prometheus GUI in PMM Server 2.12.0 (Thanks to Daniel Guzman Burgos for reporting this issue)
+* [PMM-6713](https://jira.percona.com/browse/PMM-6713): Node Summary/Nodes Overview dashboards: Show External services on dashboards
+* [PMM-7220](https://jira.percona.com/browse/PMM-7220): Keep consistent state of templates, rules, and alerts
+* [PMM-7173](https://jira.percona.com/browse/PMM-7173): VictoriaMetrics updated to v1.50.2
+
+
+
+## Bugs Fixed
+
+* [PMM-7054](https://jira.percona.com/browse/PMM-7054): ProxySQL Instance Summary dashboard: no Node Metrics
+* [PMM-7092](https://jira.percona.com/browse/PMM-7092): PMM Server Docker update from 2.11.1 to 2.12.0 leaves container in unhealthy state (Thanks to Hubertus Krogmann for reporting this issue)
+* [PMM-7208](https://jira.percona.com/browse/PMM-7208): Confusing "Access denied" message for 'Viewer' users on many dashboards
+* [PMM-6987](https://jira.percona.com/browse/PMM-6987): No IP address shown in log file of OVF appliance running in headless mode
+* [PMM-7222](https://jira.percona.com/browse/PMM-7222): Fix filters configuration
```

Comment:
```suggestion
```

Reply:
Defect, never was a released bug

Reply:
Which type should it be? @rnovikovP please change it IN JIRA ready for next jirabot run

Reply:
Was previously SKIP'd. Who unskipped it?